### PR TITLE
Move opt-in text to markdown

### LINKED ISSUE #1506: Use markdown for telemetry opt-in overlay
# Bug Report

## Summary
The link to the "More info about data we track" in this modal does not correctly open the help page. 

image

## Steps to reproduce
Open the app for the first time.

## What is the expected behaviour?
The link opens this page https://help.nexusmods.com/article/121-diagnostics-usage-data-vortex

## Other information
_(Logs, screenshots, possible fixes, expected behaviour, etc.)_

Log files can be found in these locations:

- Windows: `%LOCALAPPDATA%\NexusMods.App\Logs`
- Linux: `~/.local/state/NexusMods.App/Logs`

Please attach both `nexusmods.app.main` and `nexusmods.app.slim` log files with the most recent dates.

### PR DESCRIPTION
**Pull request by erri120**
Resolves #1506.

### CODE REVIEW

File: src/NexusMods.App.UI/Overlays/MetricsOptIn/MetricsOptInViewModel.cs [hunk 1]
```diff
@@ -1,28 +1,44 @@
...
     {
         _settingsManager = settingsManager;
+        MarkdownRendererViewModel = markdownRendererViewModel;
+
+        MarkdownRendererViewModel.Contents = """
+## Diagnostics and usage data
+
+Help us provide you with the best modding experience.
+
+With your permission, we will collect analytics information and sent it to our team to help us improve quality and performance.
```
Comment (Al12rs) [inline src/NexusMods.App.UI/Overlays/MetricsOptIn/MetricsOptInViewModel.cs:36]:
```suggestion
With your permission, we will collect analytics information and send it to our team to help us improve quality and performance.
```

Comment (Al12rs) [reply src/NexusMods.App.UI/Overlays/MetricsOptIn/MetricsOptInViewModel.cs:36]: You have to use `send` in the first sentence, and `sent` in the second sentence. 

```
With your permission, we will collect analytics information and send it to our team to help us improve quality and performance.

This information is sent anonymously and will never be shared with a 3rd party.
```

Comment (erri120) [reply src/NexusMods.App.UI/Overlays/MetricsOptIn/MetricsOptInViewModel.cs:36]: Ah right.

**Review by Al12rs — APPROVED** [review]
We should think about our Markdown translations at some point

**Comment by erri120** [comment]
> We should think about our Markdown translations at some point

Already in #1472.

**Comment by Pickysaurus** [comment]
New link: https://help.nexusmods.com/article/132-diagnostics-and-usage-nexus-mods-app

**Pull request merged**